mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -156,8 +206,18 @@ def playlistclear(context, name):
         ``playlistclear {NAME}``
 
         Clears the playlist ``NAME.m3u``.
+
+        ``NAME.m3u`` will be created if it does not exist.
```

Comment:
Since this is a comment added by us, and not a part of the docs copied from musicpd.org, you should remove one level of indentation here to get it out of the copied part.